### PR TITLE
fix err in serviceGroup doc

### PR DESCRIPTION
**Pull request by chenkaiyue**
**What type of PR is this?**

**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/components/serviceGroup.md [hunk 1]
```diff
@@ -437,28 +437,30 @@ In this example, the NodeGroup
...
-Create three clusters, one control cluster and two managed edge clusters a and B, which are registered and managed through clusternet
+### Examples
+* **Control cluster**: Cluster used to manage other clusters and create application for federation.
+* **Cluster A, Cluster B**: Two managed edge clusters, which are registered and managed through clusternet.
 
-A node in cluster A adds the label of zone: Zone1 and nodeunit Zone1; Cluster B does not join nodegroup
+Add nodes in cluster A to nodeunit zone1.
 
-Create a DeploymentGrid in the control cluster, where superedge.io/fed: "yes" is added to the labels, indicating that the DeploymentGrid needs to be distributed to the cluster, and grayscale specifies that the distributed applications use different numbers of replicas in zone1 and zone2
+Create a DeploymentGrid in the control cluster, where superedge.io/fed: "yes" is added to the labels, indicating that the DeploymentGrid needs to be distributed to managed clusters.
```
Comment (yiwei-C) [inline docs/components/serviceGroup.md:463]: Use bold text for label.

**Review by yiwei-C — CHANGES_REQUESTED** [review]
Please modify the title capitalization for all old titles as well.

File: docs/components/serviceGroup.md (hunk 1 above)
Comment (chenkaiyue) [reply docs/components/serviceGroup.md]: done~

**Review by neweyes — APPROVED** [review]
/lgtm

**Review by yiwei-C — DISMISSED** [review]

**Review by neweyes — APPROVED** [review]
/lgtm

**Pull request merged**